README.md

```diff
@@ -47,6 +47,16 @@ On the other hand, if `VERSION` hasn't been explicitly specified, CPM can automa
 `GIT_TAG` can also be set to a specific commit or a branch name such as `master` to always download the most recent version.
 The optional argument `FIND_PACKAGE_ARGUMENTS` can be specified to a string of parameters that will be passed to `find_package` if enabled (see below).
 
+A single-argument compact syntax is also supported:
+
+```cmake
+CPMAddPackage("uri@version") # A git package from a given uri with a version
+CPMAddPackage("uri@tag") # A git package from a given uri with a git tag or commit hash, or branch name
+CPMAddPackage("uri@version#tag") # A git package with both version and tag provided
+```
+
+In the shorthand syntax if the URI is of the form `user/name` or `gh:user/name`, it is interpreted as GitHub URI and converted to `https://github.com/user/name.git`. Otherwise the URI used verbatim as a git URL.
```

Comment:
Check. I can see how this default can be viewed as GitHub favoritism :)

I still think that some default is better than no default. Yes it will be difficult to change in the future, but until that future we will the most popular choice as a default. After that future, things will be undistinguishable from the no-default scenario. 

Anyway, I'll remove the gh default. We can always add it at a later point if we so decide.